res_leak7.java

```diff
@@ -0,0 +1,40 @@
+import java.sql.*;  
+class MysqlCon7{  
+   public int add(
+         int num1,
+         int num2,
+         int num3,
+         int num4,
+         int num5,
+         int num6,
+         int num7,
+         int num8,
+         int num9,
+         int num10,
+         int num11,
+         int num12,
+         int num13,
+         int num14,
+         int num15,
+         int num16,
+         int num17,
+         int num18,
+         int num19,
+         int num20
+         ){ 
+            return num1+num2+num3; +num4+num5+num6+num7+num8+num9+num10+num11+num12+num13+num14+num15+num16+num17+num18+num19+num20
+         }  
+   public static void main7(String args[]){  
+      try{  
+         Class.forName("com.mysql.jdbc.Driver");  
+         Connection con=DriverManager.getConnection(  
+               "jdbc:mysql://localhost:3306/sonoo","root","root");  
+         //here sonoo is database name, root is username and password  
+         Statement stmt=con.createStatement();  
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *stmt*. Currently, there are execution paths that do not contain closure statements. Either a) close *stmt* in a try-finally block or b) close the resource by declaring *stmt* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).